cfme/common/__init__.py

```diff
@@ -102,13 +102,6 @@ class Validatable(object):
     """
     property_tuples = []
 
-    def properties_ui(self):
```

Comment:
If you remove this method from `Validate`, what is this class's purpose without it? `validate_properties` is not going to work anymore... unless the class inheriting this class also inherits `SummaryPage` class. Which means you might as well just merge it into a single class.